tools/seed/main.go

```diff
@@ -413,6 +417,45 @@ func generateCodesAndStats(db *database.Database, realm *database.Realm) error {
 	return nil
 }
 
+// generateKeyServerStats generates stats normally gathered from a key-server. This is
+// primarily used to test statistics and graphs.
+func generateKeyServerStats(db *database.Database, realm *database.Realm) error {
+	if err := db.SaveKeyServerStats(&database.KeyServerStats{RealmID: realm.ID}); err != nil {
+		return fmt.Errorf("failed create stats config: %w", err)
+	}
+
+	year, month, day := time.Now().UTC().Date()
+	midnight := time.Date(year, month, day, 0, 0, 0, 0, time.UTC)
```

Comment:
timeutils.UTCMidnight(time.Now())

Reply:
...makes sense that there's a util for this